rally-domain/src/main/groovy/yakworks/rally/activity/model/TaskStatus.groovy

```diff
@@ -17,8 +17,6 @@ import yakworks.security.audit.AuditStamp
 @GrailsCompileStatic
 class TaskStatus implements NameCodeDescription, RepoEntity<TaskStatus>, Serializable{
 
-    String name
```

Comment:
should save these cleanups for sep PRs when possible. I know I do it too, and would rather have the clean ups then now clean ups, just keep in mind it better to try and do them in their own PR and try for that.